src/app/dialogs/install-driver/install-driver.dialog.html

```diff
@@ -1,4 +1,7 @@
-<div class="container" id="instructions">
+<h1 mat-dialog-title>{{ "CONNECTIONINSTRUCTIONS_TITLE" | translate}}</h1>
```

Comment:
This dialog is shown after selecting the robot, when we start checking for and installing prerequisites. Showing "no robot found" here might be confusing. I think we need a separate title for this install-driver dialog. It could say: "Robot Driver not found" or something.

Reply:
Thanks! I forgot to create a separate title for this dialog. I've added it now!